tests/modules/hmmer/hmmbuild/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { HMMER_HMMBUILD } from '../../../../modules/hmmer/hmmbuild/main.nf'
+
+workflow test_hmmer_hmmbuild {
+    
+    input = [
+        [ id: 'PF14720' ],      // meta map
+        file('https://raw.githubusercontent.com/nf-core/test-datasets/phyloplace/testdata/PF14720_seed.alnfaa', checkIfExists: true)
```

Comment:
Generally we prefer to put a file like this into modules, because theoretically you could restrutur ethe pipeline test data.

Also maybe there already exists a input FASTA MSA you could use (e.g. with SARS-Cov2 dataset?)? The output can be nonsense, as long as the too lruns

Reply:
Right... There are alignments in SARS-Cov2, but they are, AFAIK, very large, so would take unnecessarily long. I would prefer to move this test file to the `modules` branch.

Reply:
Ok then copying over to modules is the best approach :)

Reply:
Done. I placed it in `data/delete_me/hmmer` for lack of a better place. These aren't in the config file for tests in the modules repo, right?